packages/ckeditor5-bookmark/src/bookmarkui.ts

```diff
@@ -229,6 +235,62 @@ export default class BookmarkUI extends Plugin {
 		return formView;
 	}
 
+	/**
+	 * Creates link form menu list entry, so it'll be possible to access
+	 * the list of the bookmarks from the link form.
+	 */
+	private _registerLinkProvider() {
```

Comment:
This (along with `_scrollToBookmark`) does not seem to depend on `BookmarkUI`. Consider moving this to `BookmarkEditing`. If bookmarks are supported without ui plugin, it would still make sense to have this integration with `LinkUI`.

Reply:
Fixed in https://github.com/ckeditor/ckeditor5/pull/17593/commits/0c2beb85a1fdfbab20ae8f83df9569b4cb46ba71

Reply:
@arkflpc I'm not sure if it's a good idea, as bookmarks link provider is a part of the UI of the bookmarks. It's loosely coupled with editing itself, and the majority of things used in these providers are UI related only. 

---

I tried it here - https://github.com/ckeditor/ckeditor5/pull/17593/commits/0c2beb85a1fdfbab20ae8f83df9569b4cb46ba71, but reverted it after realizing that `BookmarkEdting` should require having `LinkUI` in dependencies which is kinda weird.